library/core/src/ptr/non_null.rs

```diff
@@ -1,13 +1,13 @@
 use crate::cmp::Ordering;
 use crate::marker::Unsize;
-use crate::mem::{MaybeUninit, SizedTypeProperties};
+use crate::mem::{self, MaybeUninit, SizedTypeProperties};
```

Comment:
This should not be needed.

Reply:
WIthout `Self ` we get the error `error[E0433]: failed to resolve: use of undeclared crate or module `mem``